packages/babel-core/test/fixtures/transformation/misc/regression-1155/expected.js

```diff
@@ -1,4 +1,6 @@
-var Foo = function (_Bar) {
+var Foo =
+/*#__PURE__*/
+function (_Bar) {
```

Comment:
This is not important... but is there a reason why this is on 3 lines instead of 1?

Reply:
its just how `babel-generator` spits out code from the AST, looks weird to me too, but 🙄

Reply:
It's probably just because we removed some logic with newlines using tokens/comments so it's like that now